Sming/project.mk

```diff
@@ -184,6 +188,11 @@ CMP_$1_LIBNAME			:= $$(COMPONENT_LIBNAME)
 CMP_$1_INCDIRS			:= $$(COMPONENT_INCDIRS)
 CMP_$1_DEPENDS			:= $$(COMPONENT_DEPENDS)
 CMP_$1_RELINK_VARS		:= $$(COMPONENT_RELINK_VARS)
+ifeq ($$(origin COMPONENT_PYTHON_REQUIREMENTS),undefined)
```

Comment:
... and `ifndef COMPONENT_PYTHON_REQUIREMENTS` ?

Reply:
That would have much clearer, but no longer necessary.